cocos/2d/CCDrawNode.cpp

```diff
@@ -398,6 +398,21 @@ void DrawNode::onDrawGLLine(const Mat4 &transform, uint32_t flags)
         // texcood
         glVertexAttribPointer(GLProgram::VERTEX_ATTRIB_TEX_COORD, 2, GL_FLOAT, GL_FALSE, sizeof(V2F_C4B_T2F), (GLvoid *)offsetof(V2F_C4B_T2F, texCoords));
     }
+
+#if CC_TARGET_PLATFORM == CC_PLATFORM_MAC || CC_TARGET_PLATFORM == CC_PLATFORM_WIN32
```

Comment:
@ricardoquesada Could you give me some suggestion on this issue? Make setLineSmooth cocos studio only, or when user call it on a mobile platform give a error message, or other? Thank you very much!